### PR TITLE
Ruowan/checklevel

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

**Review by zhenglaizhang — APPROVED** [review]

File: lib/testScenario/schemaSearcher.ts [hunk 1]
```diff
@@ -75,6 +75,13 @@ export class SchemaSearcher {
         ...this.getProperties(jsonLoader.resolveRefObj(item), jsonLoader),
       };
     });
+
+    schema.anyOf?.map((item: any) => {
```
Comment (raych1) [inline lib/testScenario/schemaSearcher.ts:79]: Is there any swagger using anyOf?

File: lib/testScenario/reportGenerator.ts [hunk 2]
```diff
@@ -429,7 +452,9 @@ export class ReportGenerator {
   }
 
   public async generateReport() {
-    this.testDefFile = await this.testResourceLoader.load(this.opts.testDefFilePath);
+    if (this.opts.testDefFilePath !== undefined) {
```
Comment (raych1) [inline lib/testScenario/reportGenerator.ts:467]: Is this test scenario file path?

File: lib/testScenario/postmanCollectionRunnerClient.ts [hunk 3]
```diff
@@ -468,7 +470,7 @@ export class PostmanCollectionRunnerClient implements TestScenarioRunnerClient {
         {
           collection: this.collection,
           environment: this.collectionEnv,
-          reporters: ["cli", "json", "junit"],
+          reporters: ["cli", "json"],
```
Comment (raych1) [inline lib/testScenario/postmanCollectionRunnerClient.ts:478]: Why remove junit?

File: lib/commands/analyze-newman-report.ts [hunk 4]
```diff
@@ -27,7 +27,7 @@ export const builder: yargs.CommandBuilder = {
     describe:
       "validation level. oav runner validate request and response with different strict level. 'valid-request' only validate request should return 2xx status code. 'default' validate both request and response.",
     string: true,
-    default: "default",
+    default: "consistency-check",
```
Comment (raych1) [inline lib/commands/analyze-newman-report.ts:30]: How about call them `valid-request` and `valid-request-response`?

File: lib/commands/run-test-scenario.ts [hunk 5]
```diff
@@ -57,9 +57,9 @@ export const builder: yargs.CommandBuilder = {
   },
   level: {
     describe:
-      "validation level. oav runner validate request and response with different strict level. 'valid-request' only validate request should return 2xx status code. 'default' validate both request and response.",
+      "validation level. oav runner validate request and response with different strict level. 'request-check' only validate request should return 2xx status code. 'consistency-check' validate both request and response.",
```
Comment (raych1) [inline lib/commands/run-test-scenario.ts:65]: Can you align to same name as above?

File: lib/testScenario/reportGenerator.ts (hunk 2 above)
Comment (ruowan) [reply lib/testScenario/reportGenerator.ts:467]: Yes

**Review by raych1 — APPROVED** [review]

**Pull request merged**